crates/forge-runner/src/running.rs

```diff
@@ -78,7 +83,8 @@ fn build_hints_dict<'b>(
 
 pub fn run_test(
     case: Arc<TestCaseRunnable>,
-    runner: Arc<SierraCasmRunner>,
+    casm_program: Arc<CairoProgram>,
+    test_details: Arc<TestDetails>,
     runner_config: Arc<RunnerConfig>,
     runner_params: Arc<RunnerParams>,
     send: Sender<()>,
```

Comment:
The exact signature of this needs refactoring, but that can be done once we're aligned on the changes to SierraCasmRunner.